client/src/pages/Profile.js

```diff
@@ -79,6 +82,10 @@ export default class Profile extends Component {
         <button className="btn" onClick={this.handleLogoutClick}>
           Me déconnecter
         </button>
+
+        <Link to="/about" className="btn btn-fw">
```

Comment:
Why a full-width button button ? There are three buttons in this page, three different sizes

Reply:
It's mainly because I copy-pasted it. It's fixed!